bip-update-process.md

```diff
@@ -0,0 +1,646 @@
+within four weeks.
+
+##### Active ↦ Abandoned
+
+A BIP may evolve from Active to Abandoned when it is no longer in active use. Any community member may initiate this
+Status update by announcing it to the mailing list, and proceed if no objections have been raised for four weeks.
+
+##### Abandoned ↦ Draft
+
+The Abandoned status is generally intended to be a final status for BIPs.
+
+If a BIP Author decides to make another attempt at a previously abandoned proposal, it is generally recommended to
+create a new proposal. (Obviously, the author may borrow any amount of inspiration or actual text from any prior BIPs as
+licensing permits.) The author should take special care to address the issues that caused the prior attempt’s
+abandonment. Even if the prior attempt had been assigned a number, the new BIP will generally be assigned a distinct
+number, unless it is obvious that the new attempt directly continues work on the same idea, in which case it may be
+reasonable to return the Abandoned BIP to Draft status.
+
+### Adoption of proposals
+
+The BIPs repository does not track the sentiment on proposals and does not track the adoption of BIPs beyond whether
+they are in active use or not. It is not intended for BIPs to list additional implementations beyond the reference
+implementation: the BIPs repository is not a signpost where to find implementations.[^OtherImplementations] After a BIP
+is advanced to Proposed, it is up to the bitcoin community to evaluate, adopt, ignore, or reject a BIP. Individual
+bitcoin projects are encouraged to publish a list of BIPs they implement. A good example of this at the time of writing
+this BIP can be observed in [Bitcoin Core’s doc/bips.md
+file](https://github.com/bitcoin/bitcoin/blob/master/doc/bips.md).
+
+### Transferring BIP Ownership
+
+It occasionally becomes necessary to transfer ownership of BIPs to a new author. In general, we would like to retain
+the original author as a co-author of the transferred BIP, but that is up to the original author. A good reason to
+transfer ownership is because the original author no longer has the time or interest in updating it or following through
+with the BIP process, or has fallen off the face of the 'net (i.e. is unreachable or not responding to email). A bad
+reason to transfer ownership is because someone doesn't agree with the direction of the BIP. We try to build consensus
+around a BIP, but if that's not possible, rather than fighting over control, the dissenters should supply a competing
+BIP.
+
+If someone is interested in assuming ownership of a BIP, they should send an email asking to take over, addressed to the
+original author and the BIP editors. If the author does not respond to email in a timely manner, the BIP editors will
+make a unilateral decision (which may be amended should the original author make a delayed reappearance).
+
+## BIP Licensing
```

Comment:
I wonder if we should also mandate a section / statement about other IP rights, if applicable. For example, I've seen this in cryptographic "competitions", e.g., CAESAR, whose goal was to find a standard for authenticated encryption:
> Intellectual property: Full disclosure of all known patents, patent applications, planned patent applications, and other intellectual-property constraints relevant to use of the cipher. This section must include the following statement: "If any of this information changes, the submitter/submitters will promptly (and within at most one month) announce these changes on the crypto-competitions mailing list." Note that intellectual-property status will be taken into account in cipher evaluation; patented submissions are permitted, but submitters are advised that patenting a cipher is likely to reduce the level of interest in the cipher.  

See https://competitions.cr.yp.to/caesar-call.html

Reply:
There could be a recommendation to use the GPLv3 or one of its succedaneous. It’s a bit of burn it with fire approach for patent issues that could grieve the free usage of bitcoin standards…so I don’t know if it’s the soundest piece of advice.

I think it would be worthy to have BIP licensing discussed openly on the mailing list in a thread of its own, rather to have this issue withholding the merge of this update to the BIP process. I guess there would be many stakeholders in the bitcoin ecosystem interested to this discussion and willing to put the ressources if there is a need to improve the current license for the BIP standards, myself included.

Reply:
Let's postpone this issue to another BIP. :P